resources/views/pages/admin/link-teacher-student.blade.php

```diff
@@ -0,0 +1,182 @@
+                <p id="chosenTeacher"></p>
+           </div>
+        </div>  
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">{{ __('linking.stap-2') }}</h3>
+            </div>
+            <div class="panel-body">
+            <div class="dropdown">
+                    <div id="myDropdown" class="dropdown-content">
+                        <input type="text" placeholder="{{ __('linking.placeholder') }}" id="myInput" onkeyup="filterFunction()">
+                        <div class="dropdown-links">
+                        @foreach($students as $student)
+                        @if($student->userlevel === 0)
+                        <a onclick="chooseStudent({{ $student->student_id }})">{{ $student->studentnr }} - {{ $student->firstname }} {{ $student->lastname }}</a>
+                        @endif
+                        @endforeach
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">{{ __('linking.stap-3') }}</h3>
+            </div>
+            <div class="panel-body">
+                <select id="selectWPLP" class="form-control" name="wplp">
+                    <option>{{ __('linking.kies-stage') }}</option> 
+                </select>
+            </div>
+        </div>
+
+        
+         {!! Form::open(array(
+            'url' =>  route('update-workplacelearningperiod')))
+            !!}
+            <div class="form-group">
+                <input id='wplp' type='text', name='wplp_id' class="form-control">
+            </div>
+                          
+            <div class="form-group">
+                <input type='text', name='teacher_id' id="teacher_id" class="form-control">
+            </div>
+            
+
+        <div class="modal-footer">
+            {{ Form::submit('Koppelen', array('class' => 'btn btn-primary')) }}
+            {{ Form::close() }}
+        </div>
+      </div>
+      
+    </div>
+  </div>
+  
+</div>
+
+<?php
+
+$allStudents = array();
+/** @var Student $student */ 
+foreach($students as $student) {
+    if($student['userlevel'] === 0){
+    array_push($allStudents, $student);
+    }
+}
+
+$allTeachers = array();
+foreach($students as $teacher) {
+    if($teacher['userlevel'] === 1){
+    array_push($allTeachers, $teacher);
+    }
+}
+
+use App\WorkPlaceLearningPeriod;
+/** @var WorkPlaceLearningPeriod $wplp */ 
+
+$wplpArray = array();
+foreach($wplperiods as $wplp) {
+    array_push($wplpArray, $wplp);
+}
+
+use App\WorkPlace;
+/** @var WorkPlace $workplace */ 
+
+$allWorkplaces = array();
+foreach($workplaces as $workplace) {
+    array_push($allWorkplaces, $workplace);
+}
+
+
+?>
+
+@include('js.linking')
+<script>
+    workplacelearningperiods = <?php echo json_encode($wplpArray) ?>;
```

Comment:
Although this is functionally correct to print JSON you might want to use the 
```blade
{!! 'asd' !!}
 ```
[functionality of Laravel's Blade](https://laravel.com/docs/5.7/blade#displaying-data). This makes it easier to separate concerns (displaying and collecting data).

Reply:
Also seems like a forgotten console log statement in there 🧐

Reply:
It won't work, I will get a Array to string conversion error.
Also `{!! $allTeachers | json_encode }` won't work.

Reply:
How about
```blade
{!! json_encode($allTeachers) !!}
```